### PR TITLE
feat(swap): Add explorer to confirmation screen and change layout

### PR DESCRIPTION
**Pull request by michalszorad**
Resolves https://emurgo.atlassian.net/browse/YOMO-850

### CODE REVIEW

File: apps/wallet-mobile/src/features/Swap/useCases/ConfirmTxScreen/ConfirmTxScreen.tsx [hunk 1]
```diff
@@ -93,9 +91,9 @@ export const ConfirmTxScreen = () => {
                 <ConfirmTx
                   wallet={wallet}
                   unsignedTx={unsignedTx}
-                  onSuccess={() => {
+                  onSuccess={(signedTx) => {
                     closeModal()
-                    navigate.submittedTx()
+                    navigate.submittedTx(signedTx.signedTx.id)
```
Comment (stackchain) [inline apps/wallet-mobile/src/features/Swap/useCases/ConfirmTxScreen/ConfirmTxScreen.tsx]: if there is animation on closeModal leverage the InteractionManager.runafter.....

File: apps/wallet-mobile/src/features/Swap/useCases/ConfirmTxScreen/ShowSubmittedTxScreen/ShowSubmittedTxScreen.tsx [hunk 2]
```diff
@@ -1,17 +1,34 @@
...
   const strings = useStrings()
   const navigate = useNavigateTo()
+  const wallet = useSelectedWallet()
+
+  const unsafeParams = useUnsafeParams()
+  const params = isParams(unsafeParams) ? unsafeParams : null
+
+  const navigateToExplorer = () => {
+    const txId = params?.txId
+    if (!isString(txId)) return
```
Comment (stackchain) [inline apps/wallet-mobile/src/features/Swap/useCases/ConfirmTxScreen/ShowSubmittedTxScreen/ShowSubmittedTxScreen.tsx]: the button is not displayed, you can drop it

Comment (michalszorad) [reply apps/wallet-mobile/src/features/Swap/useCases/ConfirmTxScreen/ShowSubmittedTxScreen/ShowSubmittedTxScreen.tsx]: `EXPLORER_URL_FOR_TX` takes only a string and the `txId` could be `undefined`

Comment (michalszorad) [reply apps/wallet-mobile/src/features/Swap/useCases/ConfirmTxScreen/ShowSubmittedTxScreen/ShowSubmittedTxScreen.tsx]: I'll add ` const txId = params?.txId ?? ''`

File: apps/wallet-mobile/src/features/Swap/useCases/ConfirmTxScreen/ConfirmTxScreen.tsx (hunk 1 above)
Comment (michalszorad) [reply apps/wallet-mobile/src/features/Swap/useCases/ConfirmTxScreen/ConfirmTxScreen.tsx]: That's now updated

File: apps/wallet-mobile/src/features/Swap/useCases/ConfirmTxScreen/ShowSubmittedTxScreen/ShowSubmittedTxScreen.tsx (hunk 2 above)
Comment (michalszorad) [reply apps/wallet-mobile/src/features/Swap/useCases/ConfirmTxScreen/ShowSubmittedTxScreen/ShowSubmittedTxScreen.tsx]: That's updated

**Review by stackchain — APPROVED** [review]

**Pull request merged**